app/Http/Controllers/HomeController.php

```diff
@@ -20,7 +23,15 @@ class HomeController extends Controller
      */
     public function index()
     {
-        return view('home', ['information' => DB::table('home_page_news')->first()->text]);
+
+        $response = Gate::inspect('view', EpistolaNews::class);
```

Comment:
Why not use `Auth::user()->can('view', EpistolaNews::class);` like at other places?